example/ios/AppcuesReactNativeExample.xcodeproj/project.pbxproj

```diff
@@ -427,6 +427,7 @@
 				);
 				MTL_ENABLE_DEBUG_INFO = YES;
 				ONLY_ACTIVE_ARCH = YES;
+				REACT_NATIVE_PATH = "${PODS_ROOT}/../../node_modules/react-native";
```

Comment:
I'm not really sure why this was added, but it happened sometime along the way during `yarn install` or `pod install` process.